BluzelleBridge/Bluzelle.NEO.Bridge.csproj

```diff
@@ -0,0 +1,62 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is it standard practice to commit .csproj files? Do we need this file in our repo?